ochk/data_source_custom_service.go

```diff
@@ -0,0 +1,110 @@
+package ochk
+
+import (
+	"context"
+	"github.com/chmurakrajowa/terraform-provider-ochk/ochk/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceCustomService() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceCustomServiceRead,
+
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"ports": {
+				Type:     schema.TypeList,
```

Comment:
tutaj ma znaczenie kolejność portów? Jeśli nie to może Set?

Reply:
Niestety ma znaczenie, ponieważ do backendu musimy odsyłać computed idki przy update. Bez matchowania elementów nie da się robić update'ów